automated/android/noninteractive-tradefed/tradefed.sh

```diff
@@ -23,13 +23,17 @@ FAILURES_PRINTED="0"
 AP_SSID=""
 # WIFI AP KEY
 AP_KEY=""
+# default to set the wifi(when required) and check the internet when not specified or set to true
+# which means it needs to specify explicitly to false to not check the internet access
```

Comment:
# By default, WiFi setup and internet access checks are enabled (INTERNET_ACCESS="true").
# To disable internet access checks, set INTERNET_ACCESS="false"

Reply:
Thanks!